README.md

```diff
@@ -116,7 +116,8 @@ selected suggestion.
 > (Temporary limitation)
 >
 > Suggestions of user-defined words may not comprise symbols that are
-> defined in some sections of the data division.
+> defined in some sections of the data division.  They do not include
```

Comment:
I really do wonder what "some sections" means... also it would be reasonable to say here something like "user-defined words from configured copybooks [link] are included" (if this is the case, what I guess).